### PR TITLE
Bug 1884547: Test PTP daemon apply match rule based on nodeLabel: use the discovered profile.

### PR DESCRIPTION
**Pull request by fedepaol**
Right now it uses the hardcoded profile, which is fine for regular mode, but not good for discovery mode.

### CODE REVIEW

**Comment by fedepaol** [comment]
/hold
still validating

File: test/ptp/ptp.go [hunk 1]
```diff
@@ -171,16 +176,17 @@ var _ = Describe("[ptp]", func() {
 
 			// 25733
 			It("PTP daemon apply match rule based on nodeLabel", func() {
-				profileSlave := "Profile Name: test-slave"
-				profileMaster := "Profile Name: test-grandmaster"
+				profileSlave := fmt.Sprintf("Profile Name: %s", slaveProfile)
+				profileMaster := fmt.Sprintf("Profile Name: %s", masterProfile)
```
Comment (SchSeba) [inline test/ptp/ptp.go:180]: nit: can you print the master profile only if it's not empty string?

**Review by SchSeba — APPROVED** [review]
/lgtm

**Pull request merged**